frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/Model.java

```diff
@@ -44,6 +45,15 @@ public Model(ModelArchive modelArchive, int queueSize) {
         // Always have a queue for data
         jobsDb.putIfAbsent(DEFAULT_DATA_QUEUE, new LinkedBlockingDeque<>(queueSize));
         failedInfReqs = new AtomicInteger(0);
+        currentWorkers = new AtomicInteger(0);
+    }
+
+    public AtomicInteger getCurrentWorkers() {
+        return currentWorkers;
+    }
+
+    public void setCurrentWorkers(AtomicInteger currentWorkers) {
```

Comment:
It doesn't seem right have a getter/setter for currentWorkers.

Reply:
True. I will remove this setter. I think get should be enough.